android/README.md

```diff
@@ -0,0 +1,177 @@
+# Introduction 
+
+This sample project covers:
+
+1. Setting up ImageKit iOS SDK
+2. Rendering images
+3. Setting authentication context for the SDK
+4. Applying common image manipulations
+5. Adding overlays to images
+6. Client-side file uploading
+
+# How to run locally
+
+This project was bootstrapped with [Android Studio](https://developer.android.com/studio/install) using [Kotlin](https://kotlinlang.org/). Head over to the link for installation if not already done.
+
+## Setup authentication
+
+In `app/src/main/java/io/imagekit/imagekitdemo/MainActivity.kt`, set the following parameters for authentication:
+
+```java
+ImageKit.init(
+   context = applicationContext,
+   publicKey = "YOUR_IMAGEKIT_PUBLIC_KEY",
+   urlEndpoint = "https://ik.imagekit.io/YOUR_IMAGEKIT_ID",
+   transformationPosition = TransformationPosition.PATH,
+   authenticationEndpoint = "http://localhost:8080/auth"
+)
+```
+
+Required parameters are `imageKitEndpoint`. The `clientPublicKey` and `authenticationEndpoint` parameter is optional and only needed if you want to use the SDK for client-side file upload. 
```

Comment:
The naming for a couple of parameter are wrong
- `urlEndpoint` not `imagekitEndpoint`
- `publicKey` not `clientPublicKey`